docs/modules/secret-operator/pages/volume.adoc

```diff
@@ -49,19 +49,33 @@ The xref:secretclass.adoc#format[format] that the secret should be written as.
 
 This can be either the default output format of the xref:secretclass.adoc#backend[backend], or a format that it defines a conversion into.
 
-=== `secrets.stackable.tech/autotls.cert.lifetime`
+=== `secrets.stackable.tech/backend.autotls.cert.lifetime`
 
 *Required*: false
 
-*Default value*: `maxCertificateLifetime` set on the SecretClass
+*Default value*: `1d`
 
 *Backends*: xref:secretclass.adoc#backend-autotls[]
 
-The lifetime the created certificate should have.
+The lifetime of the created certificate.
 Please note that you can not request a lifetime longer than allowed by `maxCertificateLifetime` on the SecretClass.
 If you do so, `maxCertificateLifetime` will be used as the certificate lifetime, to adhere to the requirements
 set by the admin that created the SecretClass.
+Format is `24h`, `1d` or `7d`.
```

Comment:
```suggestion
The format is `1h2h3m4s` (components are optional, `1m` is also a valid value).
```

Reply:
Not super happy with this formulation either, but at least it shows that they can be combined.